### PR TITLE
Some minor updates after integration test on real data

### PR DESCRIPTION
**Pull request by johanherman**
Done with a  with a Docker container. Some modifications with e.g. logging outputs, some test modifications, and process communication changes. Also makes the installation work on CentOS based
systems.

### CODE REVIEW

File: siswrap/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Idea is to have general dependencies from parent image and specific
+# dependencies here. Then for testing we install the software via
+# running the container instead on the host mounted drives.
+
+FROM arteria/dependencies:1
+LABEL Description="Test image for siswrap service" Version="0.1"
+
+# We might want to do this in a layer higher up; or add it
+RUN mkdir -pv /srv/samplesheet/processing/ && mkdir /var/log/arteria
```
Comment (johandahlberg) [inline siswrap/Dockerfile]: I agree this is a good idea since we are going to need it for all services.

File: siswrap/siswrap/siswrap.py [hunk 2]
```diff
@@ -277,26 +278,29 @@ def poll_process(self, pid):
...
-            out, err = proc.communicate()
-            wrapper.info.msg = ("Process was completed successfully with"
-                                "return code ") + str(returncode) + "."
-
-            if out is None:
-                out = "(no txt msg)"
-
-            debugmsg = "Message was: " + out
-            wrapper.info.state = ProcessInfo.STATE_DONE
+            # We can't communicate with a dead process; became obvious
```
Comment (johandahlberg) [inline siswrap/siswrap/siswrap.py]: Not sure what you mean by this? If the process is dead because it is finished?

File: siswrap/Dockerfile (hunk 1 above)
Comment (johanherman) [inline siswrap/Dockerfile]: Removed this now and added it to the dependency layer instead.

File: siswrap/siswrap/siswrap.py (hunk 2 above)
Comment (johanherman) [inline siswrap/siswrap/siswrap.py]: Yeah. Some meta data about the process is kept by Python's subprocess module, so you can e.g. poll() it several times, even after it has finished. But you can only communicate() with it once after it has finished.

**Pull request merged**